src/App.tsx

```diff
@@ -1,25 +1,52 @@
-import React from 'react';
-import './App.css';
+import React, { useState } from 'react';
+
 import { getNumbers } from './utils';
+import { Pagination } from './components/Pagination';
+import './App.css';
 
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
 const items = getNumbers(1, 42).map(n => `Item ${n}`);
 
 export const App: React.FC = () => {
+  const [currentPage, setCurrentPage] = useState(1);
+  const [perPage, setPerPage] = useState(5);
+  const totalItems = 42;
```

Comment:
`const totalItems = items.length;`
it is better to calculate it dynamically, everything happens)

Reply:
> `const totalItems = items.length;` it is better to calculate it dynamically, everything happens)

Good point